packages/wallet/src/KeyManagement/types.ts

```diff
@@ -62,7 +74,16 @@ export interface SerializableInMemoryKeyAgentData extends SerializableKeyAgentDa
   knownAddresses: GroupedAddress[];
 }
 
-export type SerializableKeyAgentData = SerializableInMemoryKeyAgentData;
+export interface SerializableLedgerKeyAgentData extends SerializableKeyAgentDataBase {
+  __typename: KeyAgentType.Ledger;
+  knownAddresses: GroupedAddress[];
```

Comment:
This can be hoisted into `SerializableKeyAgentDataBase`

Reply:
Yeah, I'd say both `extendedAccountPublicKey` and `knownAddresses` should be on base, as they both will be also used by InMemoryKeyAgent.